react/game/src/app/screens/Profile/layout.js

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Button from '@components/Form/Button';
+
+import ProfileInfo from './components/ProfileInfo';
+import ProfileEdit from './components/ProfileEdit';
+import styles from './styles.scss';
+
+function ProfileLayout({ activateEditProfile, editingProfile, profileInfo, loading }) {
+  const { backgroundPicture, profilePicture } = profileInfo;
+  return (
+    <div className={styles.profilePage}>
+      <div className={styles.profilePageContainer}>
+        {loading ? (
+          <div>Loading...</div>
+        ) : (
+          <Fragment>
+            <div
+              className={styles.profileBackground}
+              style={{
+                backgroundImage: `url(${backgroundPicture})`
+              }}
+            />
+            <div className={styles.profilePicture}>
+              <img src={profilePicture} className={styles.profilePictureImg} alt="profile" />
+            </div>
+            <div className={styles.editProfileBtnHolder}>
+              <Button
+                type="button"
+                onClick={activateEditProfile}
+                classes={styles.editProfileBtn}
+                visible={!editingProfile}
+                title="Click to edit your profile information"
```

Comment:
Try not to leave hardcoded text. Move it a variable/constant to ease translation and text changing! 😄